pages/index.js

```diff
@@ -67,7 +70,7 @@ export class Index extends Component {
               </Link>
 
               <div id="agreement" className={agreementStyle}>
-                <CdsLogo />
+                <CdsLogo className={logoStyle} />
```

Comment:
So even in this PR we're not actually downloading from the index page, so can't we just 🔥 the logo entirely?

Reply:
yes we can